compiler/rustc_mir_transform/src/inline.rs

```diff
@@ -215,6 +215,17 @@ impl<'tcx> Inliner<'tcx> {
         self.inline_call(caller_body, &callsite, callee_body);
         let new_blocks = old_blocks..caller_body.basic_blocks().next_index();
 
+        // Opportunistically remove drop terminators for types that don't need them.
```

Comment:
```suggestion
        // Opportunistically remove drop terminators for types that might
        // not need them *anymore*, after substituting generics during inlining.
```
(I was confused about what this did compared to e.g. `RemoveUnneededDrops`, initially)